README.md

```diff
@@ -44,9 +44,9 @@ module.exports = handler()
 - **functionVersion**: the [AWS Lambda][aws-lambda-url] function version that is executing
 - **serializers**: custom serializers for `err` / `error` object (based on `bunyan.stdSerializers.err`, but custom error fields, if present, are included as well), and `context` object (to prevent `log` and `child` properties from being logged) (can be overridden/extended using `options.serializers`)
 
-### context.child
+### context.child(fields, refProps)
```

Comment:
This requires such fields to be specified each time user calls `child`. I'd prefer to have this as a parameter for the middleware itself. Like:
```js
alphaLambda()
  .use(alphaLambdaBunyan({ refProps }))
```